.coveragerc

```diff
@@ -3,6 +3,8 @@ branch = True
 omit =
     # Autogenerated missed code handles other VCSes.
     devito/_version.py
+concurrency = multiprocessing
+parallel = True
```

Comment:
Note: to avoid coverage errors with mpi

Reply:
what errors?

Reply:
https://github.com/devitocodes/devito/commit/29b555b15eee3784b9d167454929e514978f41d4/checks?check_suite_id=394482751

for example